services/merak-topo/handler/topo_deploy.go

```diff
@@ -232,7 +233,7 @@ func Topo_deploy(k8client *kubernetes.Clientset, aca_image string, ovs_image str
 							SecurityContext: &sc,
 							Ports: []corev1.ContainerPort{
 								{ContainerPort: constants.AGENT_GRPC_SERVER_PORT},
-								{ContainerPort: constants.AGENT_PROMETHEUS_PORT},
+								{ContainerPort: constants.PROMETHEUS_PORT},
```

Comment:
@phudtran Please make sure this change won't break Merak-topo service.

Reply:
@zzengcs

Reply:
It shouldn't break, just a variable name change.